client-participation/config/start

```diff
@@ -0,0 +1 @@
+docker-compose up --build -d
```

Comment:
feels a bit like scope-creep. maybe leave helper scripts in another PR if you'd like to see them introduced?